### PR TITLE
Elys: Adds USDC and ATOM fee currencies

### PR DESCRIPTION
**Pull request by Wgil**
It update Elys chain definition to support different fee currencies in wallet providers.

### CODE REVIEW

File: testnets/elystestnet/assetlist.json [hunk 1]
```diff
@@ -74,6 +66,97 @@
...
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble",
+            "base_denom": "uusdc",
+            "channel_id": "channel-19"
+          },
+          "chain": {
+            "channel_id": "channel-12",
+            "path": "transfer/channel-12/uusdc"
```
Comment (JeremyParish69) [inline testnets/elystestnet/assetlist.json:97]: Neither of the IBC connections are registered. I assume you meant to actually put nolustestnet and cosmoshubtestnet ?

Comment (Wgil) [reply testnets/elystestnet/assetlist.json:97]: Just updated the names. Is it ok now? These are Noble Testnet and Cosmos Testnet IBC connections w/ Elys

Comment (Wgil) [reply testnets/elystestnet/assetlist.json:97]: Thanks for checking

Comment (JeremyParish69) [reply testnets/elystestnet/assetlist.json:97]: Thanks for correcting the name. Although neither of these IBC connections are registered here: https://github.com/cosmos/chain-registry/tree/master/testnets/_IBC

Comment (Wgil) [reply testnets/elystestnet/assetlist.json:97]: Updated. thanks

**Pull request merged**